_posts/helm-grpc.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title:  gRPC in Helm
+---
+
+Helm is the package manager for Kubernetes. Helm provides its users with a customizable mechanism for
+managing distributed applications and controlling their deployment. 
+
+I have the fortune of being a member of the phenomenal open-source Kubernetes Helm community serving as 
+a core contributor. My first day working with the Helm team was spent prototyping the architecture for
+the next generation of Helm. By the end of that day, we had procured the preliminary RPC protocol data model
+used to enable communication between Helm and it's in-cluster server component, Tiller.
+
+We chose to use protocol buffers as our data definition language (the default framework gRPC uses for
```

Comment:
We chose to use protocol buffers - the default framework gRPC uses for serialization and over-the-air transmission - as our data definition language.